src/components/SubpageUpload/upload-files.service.js

```diff
@@ -0,0 +1,21 @@
+import http from "./http-common";
```

Comment:
Put content of this file to the PageUpload and remove this file

Reply:
I dont know how to do this in a smart way, since this is how it is explained by the offical documentation.